tutorials/algorithms/04_vqd.ipynb

```diff
@@ -253,19 +254,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 31,
    "metadata": {
     "pycharm": {
      "name": "#%%\n"
     }
    },
    "outputs": [],
    "source": [
+    "from qiskit.circuit import ParameterVector\n",
+    "from qiskit.quantum_info import SparsePauliOp\n",
```

Comment:
I am not sure why the ParameterVector import is needed at all. SparsePauliOp was imported above when `H2_op` was defined and should not be needed. The change needed here is just the removal of PauliSumOp import and its usage with H2_op, which you have done. The new NumPyEigensolver can take a SparsePauliOp (which is what H2_op is) directly and while it supports PauliSumOp, for legacy reasons, that is now deprecated hence the desire for this to be updated.